cmd/tea_taskmodel.go

```diff
@@ -92,6 +92,12 @@ func (m taskModel) Update(msg tea.Msg) (taskModel, tea.Cmd) {
 		if m.spinner.ID() == msg.id {
 			m.status = msg.status
 		}
+	case fatalError:
+		if m.spinner.ID() == msg.id {
+			m.status = taskStatusError
+			m.title = fmt.Sprintf("%v: %v", m.title, msg.err)
```

Comment:
It made sense when I wrote it to replicate (the start of) the error message here. Looking at the screenshot I have doubts. Any opinion on it, @dylanratcliffe ?

Reply:
I don't live the double-pipe indentation thing TBH. A single one is okay as it mirrors terraform. Also could do with one or two less lines of whitespace I think before and after

Reply:
done.